python/pyiceberg/expressions/base.py

```diff
@@ -129,13 +129,7 @@ def bind(self, schema: Schema, case_sensitive: bool = True) -> BoundReference[T]
             BoundReference: A reference bound to the specific field in the Iceberg schema
         """
         field = schema.find_field(name_or_id=self.name, case_sensitive=case_sensitive)
-        if not field:
-            raise ValueError(f"Cannot find field '{self.name}' in schema: {schema}")
```

Comment:
I was unable to get coverage for this, because this was already handled by the `find_field` method